drivers/SmartThings/matter-switch/src/aqara-cube/init.lua

```diff
@@ -0,0 +1,217 @@
+local capabilities = require "st.capabilities"
+local clusters = require "st.matter.generated.zap_clusters"
+local device_lib = require "st.device"
+
+local cubeAction = capabilities["stse.cubeAction"]
```

Comment:
Will these capabilities remain as custom, or will they be implemented into https://github.ecodesamsung.com/iot/SmartThingsCapabilities ?

Reply:
It is already implemented in EdgeDriver for Zigbee.

Reply:
Ok I see. I glanced at that driver and was wondering if this driver should also include the following capability events in `device_added`?
`device:emit_event(cubeAction.cubeAction("noAction"))
device:emit_event(cubeFace.cubeFace("face1Up"))`

Reply:
Yes, it is simply to set the initial state.

Reply:
Sorry, I should have been more clear. I meant should those capability events be added in `device_added`? In order to set the initial state as you said. Or is that not needed in this case?

Reply:
Oh, I see. I think it only needs to be in one of `device_added` or `info_changed`. Because even when adding a device, after DEFERRED_CONFIGURE anyway, it will be a real operation, so I decided that this place would be better. If it has already been added, plugin may be updated again depending on the actual last status.

Reply:
i think it will be enough to be `info_changed` only.